flycheck.el

```diff
@@ -9217,6 +9218,9 @@ See URL `https://github.com/mivok/markdownlint'."
 See URL `https://nixos.org/nix/manual/#sec-nix-instantiate'."
   :command ("nix-instantiate" "--parse" "-")
   :standard-input t
+  :error-parser
+  (lambda (output checker buffer)
+    (flycheck-parse-with-patterns (ansi-color-filter-apply output) checker buffer))
```

Comment:
I wonder whether it's best to add an autoload or to just require `'ansi-color` in that lambda.

Reply:
fyi the `erlang-rebar3` checker (which does exactly the same as this parser) has `(require 'ansi-color)`, so you may want to change them together for consistency's sake